blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractPipelineCreateRequest.java

```diff
@@ -72,4 +72,8 @@ protected ModifiableTopLevelItemGroup getParent() {
 
         return parent;
     }
+
+    protected BlueOrganization findOrganization() {
+        return OrganizationFactory.getInstance().get(getOrganization());
```

Comment:
@alvarolobato this code was written by @vivek - I am not sure if this is correct or not.